elk/plotting/command.py

```diff
@@ -0,0 +1,42 @@
+import shutil
+from dataclasses import dataclass
+
+import rich
+from rich.panel import Panel
+from simple_parsing import field
+
+from ..files import sweeps_dir
+from .visualize import visualize_sweep
+
+
+def pretty_error(msg):
+    rich.print(Panel(f"[red]Error[/red]: {msg}"))
+
+
+@dataclass
+class Plot:
+    sweeps: list[str] = field(positional=True, default_factory=list)
+    overwrite: bool = False
+
+    def execute(self):
+        sweeps_root_dir = sweeps_dir()
+        sweep = max(sweeps_root_dir.iterdir(), key=lambda f: f.stat().st_ctime)
+        if self.sweeps and not (sweeps_root_dir / self.sweeps[0]).exists():
+            pretty_error(
+                f"No sweep with name {{{self.sweeps[0]}}} found in {sweeps_root_dir}"
+            )
+        elif len(self.sweeps) > 1:
+            # TODO support more than one sweep
+            pretty_error(
+                f"""{len(self.sweeps)} paths specified.
+                Only one sweep is supported at this time."""
+            )
+        elif (sweep / "viz").exists() and not self.overwrite:
+            pretty_error(
+                f"[blue]{sweep / 'viz'}[/blue] already exists. "
+                f"Use --overwrite to overwrite."
+            )
+        else:
+            if self.overwrite:
+                shutil.rmtree(sweep / "viz")
+            visualize_sweep(sweep)
```

Comment:
This doesn't actually use the value of `self.sweeps` at all, so this command _only_ supports visualizing the most recent sweep right now
<img width="870" alt="Captura de pantalla 2023-05-22 a la(s) 11 53 53 a m" src="https://github.com/EleutherAI/elk/assets/39116809/c2da9a7a-f08f-43ab-a5f5-af29791195cd">

Reply:
Fixed